src/common/util.js

```diff
@@ -0,0 +1,11 @@
+const getCompiler = (path) => {
+  let i = 0;
+  let elderNode;
+  do {
+    elderNode = path.getParentNode(i);
+    i += 1;
+  } while (elderNode.type !== 'SourceUnit');
```

Comment:
This assumes that `SourceUnit` will always be the root node, which I think is a fair assumption. But I'd rather codify this assumption:

```
} while (elderNode.type && elderNode.type !== 'SourceUnit')

if (!elderNode || !elderNode.compiler) {
  throw new Error("AssertionError: expected elderNode.compiler to exist")
}

Reply:
we don't use this anymore